packages/mobile/src/home/Home.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
-import { SafeAreaView, Text } from 'react-native';
+import { Text, View } from 'react-native';
+import Header from '../components/Header';
 
 const Home = ({ navigation }) => {
   return (
-    <SafeAreaView>
+    <View style={{ flex: 1, backgroundColor: '#f5f5f5' }}>
+      {/* header component that is 10% of view port */}
+      <View style={{ flex: 1 }}>
+        <Header />
+      </View>
+
+      {/* body that is 90% of view port */}
+      <View style={{ flex: 9 }} />
```

Comment:
Confused why there's empty `View` container here. The header component doesn't need to be sticky if that's what you're worried about.